.github/workflows/tf-plan-hash.yml

```diff
@@ -0,0 +1,130 @@
+name: HASH Terraform Plan
+
+# TODO: While this workflow is specific to HASH, we should consider
+#  making this a reusable workflow, parameterized on the directory and environment.
+
+on:
+  workflow_dispatch:
+    inputs:
+      env:
+        type: choice
+        options: ["prod"]
+        description: "Environment to plan"
+        required: true
+        default: "prod"
+
+  pull_request:
+    branches:
+      - main
```

Comment:
Why do we only run it against main?

Reply:
For `pull_request` this means that it only runs on PRs targetting `main`

Reply:
Removed in ef00b5449bb9f2b455fd2c2f2065afacef697c0c, thanks :)